vernac/declare.mli

```diff
@@ -289,8 +289,8 @@ module Proof : sig
       instead *)
   type proof_object
 
-  val close_proof : opaque:Vernacexpr.opacity_flag -> keep_body_ucst_separate:bool -> t -> proof_object
-  val close_future_proof : feedback_id:Stateid.t -> t -> closed_proof_output Future.computation -> proof_object
+  val stm_close_proof : opaque:Vernacexpr.opacity_flag -> keep_body_ucst_separate:bool -> t -> proof_object
+  val stm_close_future_proof : feedback_id:Stateid.t -> t -> closed_proof_output Future.computation -> proof_object
```

Comment:
since these APIs are specific for the stm, I did rename them so that you know it (and don't call them) if you are working on a non-stm coq toplevel

Reply:
shouldn't you then also rename the ones in vernacstate?

Reply:
Why are they specific to the stm ? What should a non-stm toplevel call instead?

[instead of the renaming you can also just wrap them in a `Stm` module for namespacing]

Reply:
This whole module is deprecated, I believe you did deprecate it.
IMO a regular toplevel should just vernacinterp the Qed or Defined, and not fiddle with proofs directly.